frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -265,18 +266,23 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
     setUnfilteredNodes(setSelectedUnfilteredNodes);
   };
 
-  const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
-    return selectedNodes.map((node: NodeKind) => {
-      const patch = [
-        {
-          op: 'add',
-          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
-          value: '',
-        },
-      ];
-      return k8sPatch(NodeModel, node, patch);
-    });
-  };
+  const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] =>
+    const patch = [
```

Comment:
It should be written inside function body i.e. `{` and `}` and return the reduce statement

Reply:
It's not a huge deal, but I don't feel like it's necessary to declare this variable inside the scope of the reducer. It's a static value. There's no need to re-declare the same variable on every iteration over the list of nodes.

Reply:
It was a nit, but I asked for this variable to be declared at a higher scope.